contribs/gmf/less/displayquerywindow.less

```diff
@@ -89,21 +93,12 @@
       }
     }
   }
-  .gmf-displayquerywindow-header {
-    height: @displayquerywindow-detailed-header-height;
-    margin-bottom: @app-margin;
-    p {
-      margin-bottom: @app-margin;
-    }
-  }
   .gmf-displayquerywindow-title {
     font-weight: bold;
   }
-  .gmf-displayquerywindow-subtitle {
-    min-height: 1em;
-  }
   .gmf-displayquerywindow-details {
-    height: @displayquerywindow-detailed-details-height;
+    display: none;
+    flex: 1;
```

Comment:
The line upper is `display none`... so, this css do somethings ?
Don't you want to use `visibility:hidden` ?